docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -294,6 +406,19 @@ After copying all the files, invoke the import script:
 docker exec testneo4j bin/cypher-shell -u neo4j -p ${NEO_PASS} -f import.cypher
 ----
 
+[TIP]
+====
+If the Neo4j import fails, you might need to delete leftover data before importing it again.
```

Comment:
killing and rerunning the `testneo4j` container might be simpler/safer.

Reply:
Ah yes, good point 😅